build.rs

```diff
@@ -1,3 +1,38 @@
+/// This build script checks for env vars to build ouch with shell completions.
+///
+/// # How to generate shell completions:
+///
+/// Set `OUCH_COMPLETIONS_FOLDER` to the name of the destination folder:
+///
+/// ```sh
+/// OUCH_COMPLETIONS_FOLDER=my-folder cargo build
+/// ```
```

Comment:
@jcgruenhage sorry for the delay on this, I'm trying to get back to ouch.

The final code ended up being a bit different from what we agreed, I maintained retrocompability to keep older packages of ouch working.

Everything is now documented in this file.

Reply:
@jcgruenhage will you need to update something in the new void package?

Reply:
Looks good to me. And yeah, will need adjustment once a new release of ouch is out. Both of the patches can be dropped and the pre_build phase will need to be adjusted to the new way these env vars should be working, but that shouldn't be too much work.

cc: @ElDifinitivo